backend/helper_zero/views/auth.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import status, viewsets
+from rest_framework.response import Response
+
+from helper_zero.models import Organization, DonationRequest
+from helper_zero.serializers import OrganizationSerializer, DonationRequestSerializer
+
+class AuthView(viewsets.ViewSet):
+  def list(self, request):
+    auth_token = request.GET.get('auth_token')
+    org = Organization.objects.filter(auth_token=auth_token).first()
+    if org == None:
+      return Response([])
```

Comment:
Should we return a Response with a 4XX code here?

Reply:
hmmmmm i was thinking that the frontend should just deal with what happens when no org comes back, like maybe it would just create an account for the org. so it's not necessarily an error